### PR TITLE
Create subclasses of CreateMap and CreatePlot

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
This PR addresses issue #49.

### CODE REVIEW

File: src/emcpy/plots/create_plots.py [hunk 1]
```diff
@@ -1,14 +1,150 @@
...
+import cartopy.feature as cfeature
 import matplotlib.pyplot as plt
+from matplotlib import rcParams
 from matplotlib.colors import Normalize
 from scipy.interpolate import interpn
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from emcpy.plots.map_tools import Domain, MapProjection
 from emcpy.stats import get_linear_regression
 
+__all__ = ['EMCPyPlots', 'CreatePlot', 'CreateMap']
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/create_plots.py]: Would a user ever call the base class? If not, should `EMCPyPlots` be removed here?

File: src/emcpy/plots/create_plots.py [hunk 2]
```diff
@@ -1,14 +1,150 @@
...
-class CreatePlot:
+    def __init__(self):
+        return
+
+    def add_title(self, label,
+                  loc='center',
+                  fontsize=12,
+                  fontweight='normal',
+                  color='k',
+                  va='baseline'):
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/create_plots.py]: can we keep this as `verticalalignment`? `va` means nothing to me besides Virginia or 'go' in Spanish

**Review by CoryMartin-NOAA — COMMENTED** [review]
Couple of small comments

File: src/emcpy/plots/create_plots.py [hunk 3]
```diff
@@ -315,67 +391,198 @@ def set_yticklabels(self, labels=list(),
...
 
-    def add_colorbar(self, label=None, orientation='horizontal',
-                     label_fontsize=12, extend='neither'):
+
+class CreateMap(EMCPyPlots):
+    """
+    Creates a map axes on a figure to plot data.
+    """
+
+    def __init__(self, fig=plt.figure(figsize=(12, 8)),
```
Comment (aerorahul) [inline src/emcpy/plots/create_plots.py]: Is there a reason, `fig=plt.figure(figsize=(12,8))` not be in the `EMCpyPlots` constructor?
Same for `CreatePlots`.
It seems an appropriate place for initializing a figure.

File: src/emcpy/plots/create_plots.py [hunk 4]
```diff
@@ -1,14 +1,150 @@
...
 from emcpy.stats import get_linear_regression
 
+__all__ = ['CreatePlot', 'CreateMap']
 
-__all__ = ['CreatePlot']
 
+class EMCPyPlots:
 
-class CreatePlot:
+    def __init__(self):
```
Comment (aerorahul) [inline src/emcpy/plots/create_plots.py]: I like this, but I think this default constructor should return a figure handle via `fig=plt.figure(figsize=(12,8))`. One could provide a `figsize` as input argument if necessary/

**Comment by kevindougherty-noaa** [comment]
@aerorahul @CoryMartin-NOAA I made the default constructor where the figure is created and changed `CreateMap` and `CreatePlot` to have an input argument as `figsize`.

**Review by CoryMartin-NOAA — APPROVED** [review]
I think this looks good to me

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by kevindougherty-noaa** [comment]
Closes #49

**Pull request merged**